lib/ngx/errlog.lua

```diff
@@ -21,6 +21,8 @@ int ngx_http_lua_ffi_errlog_set_filter_level(int level, unsigned char *err,
     size_t *errlen);
 int ngx_http_lua_ffi_errlog_get_msg(char **log, int *loglevel,
     unsigned char *err, size_t *errlen);
+
+int ngx_http_lua_ffi_errlog_log_level(ngx_http_request_t *r);
```

Comment:
Better use verbs in the function names, like `ngx_http_lua_ffi_errlog_get_sys_filtering_level`. Also, we should emphasize that this is the system default filtering level since we already have the `set_filter_level` function to set a "soft filtering level".